llvm/include/llvm/Transforms/Utils/Cloning.h

```diff
@@ -194,6 +194,15 @@ void CloneFunctionMetadataInto(Function *NewFunc, const Function *OldFunc,
                                ValueMapTypeRemapper *TypeMapper = nullptr,
                                ValueMaterializer *Materializer = nullptr);
 
+/// Clone OldFunc's body into NewFunc.
+void CloneFunctionBodyInto(Function *NewFunc, const Function *OldFunc,
```

Comment:
Can `NewFunc` / `OldFunc` be nullptr? If not, they should be references

Reply:
Thanks! Updated them to be references.